tests/e2e/e2e_common.go

```diff
@@ -175,7 +175,7 @@ const (
 	resizerContainerName                      = "csi-resizer"
 	scParamDatastoreURL                       = "DatastoreURL"
 	scParamFsType                             = "csi.storage.k8s.io/fstype"
-	scParamStoragePolicyID                    = "StoragePolicyId"
+	scParamStoragePolicyID                    = "storagePolicyId"
```

Comment:
we should do same thing for `scParamDatastoreURL` and `scParamStoragePolicyName`?